app/src/androidTest/java/com/github/orkest/ViewModel/profile/ProfileViewModelTest.kt

```diff
@@ -242,5 +245,5 @@ class ProfileViewModelTest {
         try { viewModel.unfollow() } catch(e: java.lang.IllegalArgumentException){
             assertEquals("Cannot call this function when visiting the current logged-in user's profile", e.message)
         }
-    }
+    }**/
 }
```

Comment:
The test method names should reflect the updated method names and the new test logic. For example, followFunctionalityUpdatesUsers should be renamed to something like updateCurrentUserFollowingsAndUserFollowers.

The commented-out test methods, followThrowsExceptionWhenCalledInCurrentProfile and unFollowThrowsExceptionWhenCalledInCurrentProfile should either be removed or updated to work with the new implementation and properly handle the exception scenarios.